src/plugins/data/public/search/session/session_service.test.ts

```diff
@@ -30,9 +30,19 @@ describe('Session service', () => {
 
   beforeEach(() => {
     const initializerContext = coreMock.createPluginInitializerContext();
+    const startService = coreMock.createSetup().getStartServices;
     sessionService = new SessionService(
       initializerContext,
-      coreMock.createSetup().getStartServices,
+      () =>
+        startService().then(([coreStart, ...rest]) => [
+          {
+            ...{
```

Comment:
nit: not sure that it has some sense especially in tests. think it should be replaced to: 
```
 startService().then(([coreStart, ...rest]) => [
    {
      ...coreStart,
      application: { ...coreStart.application, currentAppId$: new BehaviorSubject('app') },
    },
    ...rest,
  ]),
```